shared/rakefiles/xk_infra.rake

```diff
@@ -98,22 +98,16 @@ task :apply_common_infra => [@gcp_creds_file] do
   # permssions to an operator. The following code checks that the SA has the
```

Comment:
Most of this comment (L87-100) is now documented elsewhere. Can you edit it down or delete it?